### PR TITLE
Update all gems including Poltergeist and Rspec.

### PR DESCRIPTION
**Pull request by jwreachlocal**
Since Rspec is being updated, some of the tests need to be corrected since be_true and be_false are not longer valid.

### CODE REVIEW

**Comment by clarkbremer** [comment]
:hammer:

File: spec/lib/snapdragon/suite_spec.rb [hunk 1]
```diff
@@ -113,9 +113,7 @@
   end
 
   describe "#require_paths" do
-    it "returns the merged set of the require paths of each spec file" do
-      pending
-    end
+    it "returns the merged set of the require paths of each spec file"
```
Comment (clarkbremer) [inline spec/lib/snapdragon/suite_spec.rb]: You could say: `pending "returns the ....."`

**Comment by clarkbremer** [comment]
:+1:

File: spec/lib/snapdragon/suite_spec.rb (hunk 1 above)
Comment (jwreachlocal) [inline spec/lib/snapdragon/suite_spec.rb]: yes, but i didn't want to change what was there.

**Pull request merged**